api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentTooltipTemplate.md

```diff
@@ -30,370 +30,14 @@ Specifies whether you click a button or an appointment element.
-            const isDeleteButtonExist = !isAppointmentDisabled && isDeleteAllowed;
-
-            const onDeleteButtonClick = (e) => {
-                scheduler.instance.deleteAppointment(props.appointmentData);
-                e.event.stopPropagation();
-                scheduler.instance.hideAppointmentTooltip();
-            };
-
-            return (
-                <Tooltip
-                    {...props}
-                    isDeleteButtonExist={isDeleteButtonExist}
-                    onDeleteButtonClick={onDeleteButtonClick}
-                    color={color}
-                />
-            );
-        }, []);
-
-        render(
-            <Scheduler ...
-                ref={schedulerRef}
-                appointmentTooltipRender={appointmentTooltip}
-            >
-            </Scheduler>
-        );
-    }
-
-    export default App;
-
-    <!-- tab: Tooltip.js -->
-    import React, { useCallback } from 'react';
-
-    const  Tooltip = (props) => {
-        const onDeleteButtonClick = useCallback((e) => {
-            props.onDeleteButtonClick(e);
-        }, [props.onDeleteButtonClick]);
-
-        return (
-            <div className={"dx-tooltip-appointment-item"}>
-                <div className={"dx-tooltip-appointment-item-marker"}>
-                    <div
-                        className={"dx-tooltip-appointment-item-marker-body"}
-                        style={{ backgroundColor: props.color }}
-                    ></div>
-                </div>
-                <div className={"dx-tooltip-appointment-item-content"}>
-                    <div className={"dx-tooltip-appointment-item-content"}>
-                        <div className={"dx-tooltip-appointment-item-content-subject"}>
-                            {props.appointmentData.text}
-                        </div>
-                        <div className={"dx-tooltip-appointment-item-content-date"}>
-                            {props.appointmentData.startDate.toString()}
-                        </div>
-                    </div>
-                </div>
-
-                {props.isDeleteButtonExist ? (
-                    <div className={"dx-tooltip-appointment-item-delete-button-container"}>
-                        <Button
-                            className={"dx-tooltip-appointment-item-delete-button"}
-                            icon="trash"
-                            stylingMode="text"
-                            onClick={onDeleteButtonClick}
-                        ></Button>
-                    </div>
-                ) : (
-                    <></>
-                )}
-            </div>
-        );
-    }
-
-    export default Tooltip;
-
----
+If you want to change tooltips appearance, implement a custom template with the `appointmentTooltipTemplate` property. Refer to the [Customize Appointment Tooltip](/Documentation/Guide/UI_Components/Scheduler/Appointments/Customize_Appointment_Tooltip/) section for details.
```

Comment:
If you want to change **tooltip** appearance OR If you want to change **tooltips'** appearance